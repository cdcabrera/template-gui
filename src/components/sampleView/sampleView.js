import React from 'react';
import { PageSection } from '@patternfly/react-core';
import { connectRouter } from '../../redux';
import SampleCard from '../sampleCard/sampleCard';

class SampleView extends React.Component {
  componentDidMount() {}

  render() {
    return (
      <React.Fragment>
        <PageSection>
          <h1>Sample view</h1>
          <SampleCard />
        </PageSection>
      </React.Fragment>
    );
  }
}

SampleView.propTypes = {};

SampleView.defaultProps = {};

const mapStateToProps = () => ({});

const mapDispatchToProps = () => ({});

const ConnectedSampleView = connectRouter(mapStateToProps, mapDispatchToProps)(SampleView);

export { ConnectedSampleView as default, ConnectedSampleView, SampleView };
