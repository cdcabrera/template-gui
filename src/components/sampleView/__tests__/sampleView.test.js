import React from 'react';
import { shallow } from 'enzyme';
import { SampleView } from '../sampleView';

describe('SampleView Component', () => {
  it('should render a non-connected component', () => {
    const props = {};

    const component = shallow(<SampleView {...props} />);

    expect(component).toMatchSnapshot('non-connected');
  });
});
