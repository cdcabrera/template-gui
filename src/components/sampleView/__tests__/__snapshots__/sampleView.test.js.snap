// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SampleView Component should render a non-connected component: non-connected 1`] = `
<Fragment>
  <PageSection
    className=""
    noPadding={false}
    noPaddingMobile={false}
    variant="default"
  >
    <h1>
      Sample view
    </h1>
    <Connect(SampleCard) />
  </PageSection>
</Fragment>
`;
